lib/hexpm/web/plugs.ex

```diff
@@ -81,6 +81,9 @@ defmodule Hexpm.Web.Plugs do
     param_set? = Map.has_key?(conn.params, "repository")
 
     case conn.path_info do
+      ["api", "packages"] ->
+        conn
+
```

Comment:
I'm not a fan of this function explicitly checking routes but I don't know of a better way of doing it.